doc/sphinx/source/recipes/recipe_gier20bg.rst

```diff
@@ -0,0 +1,231 @@
+
+#. Script xco2_analysis/sat_masks.ncl:
+
+    * Optional diag_script_info attributes:
+        * ``output_file_type``: output file type for plots. Default: png
+        * ``var_plotname``: String formatting how variable should be named in plots
+          defaults to short_name if not assigned
+        * ``c3s_plots``: Missing value plots seperated by timeseries of c3s satellites
+
+#. Script xco2_analysis/station_comparison.ncl:
+
+    * Required diag_script_info attributes:
+        * ``var_order``: in this case xco2, co2, co2s - column averaged with obs dataset
+          first, then 2D variable, followed by surface stations
+
+    * Optional diag_script_info attributes:
+        * ``output_file_type``: output file type for plots. Default: png
+        * ``var_plotnames``: String formatting how variables should be named in plots
+          defaults to short_name if not assigned
+        * ``overwrite_altitudes``: Give other altitude values than the ones attached in
+          the station data. Valid if altitude changes and
+          timeseries spans range with different sample
+          altitude. Caveat: If used, need to give altitude
+          values for all stations.
+        * ``output_map``: boolean if stations to be displayed on map. As this requires
+          finetuning, currently only implemented for station set of
+          (ASK, CGO, HUN, LEF, WIS) following the paper. Change for different
+          plot inset locations, if others are desired.
+
+Variables
+---------
+
+* *xco2* (atmos, monthly, longitude, latitude, time)
+* *co2s* (atmos, monthly, longitude, latitude, time)
+* *co2* (atmos, monthly, pressure, longitude, latitude, time)
+* *tas* (atmos, monthly, longitude, latitude, time)
+* *tasa* (atmos, monthly, longitude, latitude, time)
+
+
+Observations and reformat scripts
+---------------------------------
+
+* CDS-XCO2_ (*xco2*)
+* ESRL_ (*co2s*)
+* GISTEMP_ (*tasa*)
+* MODIS_ (land cover map, auxiliary data folder)
+
+.. _ESRL: https://www.esrl.noaa.gov/gmd/dv/data/
+.. _GISTEMP: https://data.giss.nasa.gov/gistemp/
+.. _CDS-XCO2: https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-carbon-dioxide?tab=form
+.. _MODIS: https://daac.ornl.gov/cgi-bin/dsviewer.pl?ds_id=968
+
+References
+----------
+
+* Gier, B. K., Buchwitz, M., Reuter, M., Cox, P. M., Friedlingstein, P.,
+  and Eyring, V.: Spatially resolved evaluation of Earth system models with
+  satellite column-averaged CO2, Biogeosciences, 17, 6115–6144,
+  https://doi.org/10.5194/bg-17-6115-2020, 2020.
+
+Example plots
+-------------
+
+.. _fig_gier20bg_1:
+.. figure::  /recipes/figures/gier20bg/fig01.png
+   :align:   center
+   :width: 80%
+
+   Mean fractional coverage of monthly satellite data.
+
+.. _fig_gier20bg_2:
+.. figure::  /recipes/figures/gier20bg/fig02.png
+   :align:   center
+   :width: 80%
+
+   Comparison of time series from satellite, in situ, and models sampled
+   accordingly. Caveat: inset plot positions are hardcoded.
```

Comment:
```suggestion
   accordingly.
```

I don't think you need to apologize here :wink:

Reply:
How about changing it to a "note"? I'd like to keep the mention of it being hardcoded in, so that if people would like to use different stations they are aware that it's possible if they change the placement themselves.

Reply:
Fine for me if you want to keep it :+1: